### PR TITLE
Misc Polish

### PR DESCRIPTION
**Pull request by alexpaxton**
- [x] CHANGELOG.md updated
  - [x] Rebased/mergable
  - [x] Tests pass

Connect #1455 
Connect #1423 
Connect #1426 

### The problem
- Some pages do not use the page spinner and instead use `Loading...`
- Empty state of rules table is not very helpful, is a UX dead end
- In sources table, it is hard to tell which source you are connected to because all sources offer a `Connect` option
- The UI copy `DB Management` is inconsistent
- Query placeholder in rule builder page does not look like a placeholder
- The instructional `Enter values below` block in CSV type tempvars takes up a lot of space

### The Solution
- Fixes for all of the aforementioned, see below

### Preview
screen shot 2017-05-11 at 5 15 08 pm
screen shot 2017-05-11 at 5 30 28 pm
screen shot 2017-05-11 at 5 30 43 pm
screen shot 2017-05-11 at 5 32 06 pm
screen shot 2017-05-11 at 5 34 00 pm

### CODE REVIEW

**Comment by alexpaxton** [comment]
kap-rule-builder-placeholder

**Comment by alexpaxton** [comment]
proper-sort-arrows

**Comment by alexpaxton** [comment]
screen shot 2017-05-11 at 5 36 48 pm

File: ui/src/alerts/components/AlertsTable.js [hunk 1]
```diff
@@ -82,68 +82,83 @@ const AlertsTable = React.createClass({
       <div className="panel panel-minimal">
         <div className="panel-heading u-flex u-ai-center u-jc-space-between">
           <h2 className="panel-title">{this.props.alerts.length} Alerts</h2>
-          <SearchBar onSearch={this.filterAlerts} />
+          {this.props.alerts.length > 0
```
Comment (lukevmorris) [inline ui/src/alerts/components/AlertsTable.js]: `this.props.alerts.length`

File: ui/src/alerts/components/AlertsTable.js [hunk 2]
```diff
@@ -82,68 +82,83 @@ const AlertsTable = React.createClass({
...
-              </tr>
-            </thead>
-            <tbody>
-              {alerts.map(({name, level, time, host, value}) => {
-                return (
-                  <tr key={`${name}-${level}-${time}-${host}-${value}`}>
-                    <td className="monotype">{name}</td>
-                    <td
-                      className={`monotype alert-level-${level.toLowerCase()}`}
+          {this.props.alerts.length > 0
```
Comment (lukevmorris) [inline ui/src/alerts/components/AlertsTable.js]: `this.props.alerts.length`

File: ui/src/alerts/components/AlertsTable.js (hunk 1 above)
Comment (alexpaxton) [inline ui/src/alerts/components/AlertsTable.js]: Done

File: ui/src/alerts/components/AlertsTable.js (hunk 2 above)
Comment (alexpaxton) [inline ui/src/alerts/components/AlertsTable.js]: Done

**Pull request merged**